FrontPage/API.swift

```diff
@@ -188,8 +205,14 @@ public final class CreateTaskMutation: GraphQLMutation {
 }
 
 public final class DeleteSubscription: GraphQLSubscription {
+  /// subscription delete {
+  ///   taskDeleted {
+  ///     __typename
+  ///     ...TaskFields
+  ///   }
+  /// }
```

Comment:
is this a generated file? Do we actually need these commented code? If we do, let's uncomment them, otherwise we should just remove them

Reply:
This file is a generated file and is regenerated every time a build is made.